### DISCUSSION TITLE
Incorrect port on compose and Docker run docs

### DISCUSSION THREAD

**Discussion by mcapuccini** [open-webui/open-webui#4213] [General]
# Bug Report

## Description

**Bug Summary:**
You are forwarding the wrong port on compose and docker run i the docs. For example, in your docs you have:

```
docker run -d -p 3000:8080 -v ollama:/root/.ollama -v open-webui:/app/backend/data --name open-webui --restart always ghcr.io/open-webui/open-webui:ollama
```

The correct command should be:
```
docker run -d -p 8080:8080 -v ollama:/root/.ollama -v open-webui:/app/backend/data --name open-webui --restart always ghcr.io/open-webui/open-webui:ollama
```

Perhaps you changed the uvicorn port in Docker at a certain point. Anyhow updating documentation and Docker compose should fix this. Alternatively you could configure uvicorn to port 3000.

**Comment by justinh-rahb** [comment]
No, the docs are correct. I'm afraid it's you who is mistaken. Please refer to Docker documentation regarding "Published Ports".
